src/read_resume_data.cpp

```diff
@@ -135,11 +135,6 @@ namespace {
 				{
 					ec = err;
 				}
-				else if (((ret.info_hashes.has_v1() && ret.info_hashes.v1 != ret.ti->info_hashes().v1)
-					|| (ret.info_hashes.has_v2() && ret.info_hashes.v2 != ret.ti->info_hashes().v2)))
```

Comment:
@glassez this is the same check as on line 128 (but inverted), so I moved the block to its else-block

Reply:
:+1:
I didn't even notice that info hash is calculated before torrent info is parsed.